GliaWidgetsTests/CallVisualizer/VideoCall/VideoCallTests.swift

```diff
@@ -0,0 +1,90 @@
+import XCTest
+@testable import GliaWidgets
+
+final class VideoCallTests: XCTestCase {
+    func test_vc_deinit() {
+        weak var weakViewController: CallVisualizer.VideoCallViewController?
+        autoreleasepool {
+            let viewController = CallVisualizer.VideoCallViewController(
+                props: .init(videoCallViewProps: .mock),
+                environment: .mock
+            )
+            weakViewController = viewController
+        }
+        XCTAssertNil(weakViewController, "VisitorCodeViewController not deinitilized")
+    }
```

Comment:
Here it would be good to add a space after the closing bracket to separate it from the new test

Reply:
fixed [3c6d82a](https://github.com/salemove/ios-sdk-widgets/pull/496/commits/3c6d82aa1dae1b906a78bffc8a43018b3ad75965)